src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const totalPrice = days * pricePerDay;
```

Comment:
let's use upperCase for variables
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/91946036/f2367a9f-2bce-41f6-a68e-d46b2b3614a6)